infra/aws.conf

```diff
@@ -172,6 +172,7 @@ plugins {
         enabled = true
         use-default-credential-provider = true
         default-bucket = "nexus-delta-production"
+        prefix = ""
```

Comment:
Do we know the value for this yet?

Reply:
It should be the same as the one defined for the storage service